opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/wcaro/ANCVisitTests.java

```diff
@@ -0,0 +1,175 @@
+package org.smartregister.chw.activity.wcaro;
+
+import android.Manifest;
+
+import androidx.test.filters.LargeTest;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.smartregister.chw.R;
+import org.smartregister.chw.activity.LoginActivity;
+import org.smartregister.chw.activity.utils.Configs;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Order;
+import org.smartregister.chw.activity.utils.OrderedRunner;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withSubstring;
+
+
+@LargeTest
+//@RunWith(AndroidJUnit4.class)
+@RunWith(OrderedRunner.class)
+public class ANCVisitTests {
+    @Rule
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Rule
+    public GrantPermissionRule permissionRule = GrantPermissionRule.grant(android.Manifest.permission.CALL_PHONE);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule1 = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+
+    private Utils utils = new Utils();
+
+    public void setUp() throws InterruptedException {
+        utils.logIn(Constants.WcaroConfigUtils.wCaro_username, Constants.WcaroConfigUtils.wCaro_password);
+    }
+
+    @Order(order = 1)
+    @Test
+    public void testAncVisit() throws Throwable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)